proxy/src/main/java/com/velocitypowered/proxy/connection/client/LoginSessionHandler.java

```diff
@@ -225,6 +237,12 @@ private void initializePlayer(GameProfile profile, boolean onlineMode) {
   }
 
   private void finishLogin(ConnectedPlayer player) {
+    Optional<RegisteredServer> toTry = player.getNextServerToTry();
+    if (!toTry.isPresent()) {
+      player.disconnect(VelocityMessages.NO_AVAILABLE_SERVERS);
```

Comment:
You forgot this case

Reply:
i think this should not be in this pr. https://github.com/VelocityPowered/Velocity/commit/d2b8271eb4b23578d833058af4e2fb1cbff4ba94#diff-8e51e6774e951b58c22066197a838ad4

Reply:
Agreed. This reverts a feature added in 1.1.0 to allow users to configure the proxy to run without servers configured. Please revert this.